### ISSUE TITLE
filterAll is not a function 

### ISSUE THREAD

**Issue by shovondu86** [dc-js/dc.datatables.js#5]
![dctable](https://user-images.githubusercontent.com/7312809/47782029-6dc7ac00-dd29-11e8-9a96-9138770dafaf.PNG)

**Comment by gordonwoodhull** [comment]
Thanks @shovondu86! It's easy to lose track of all the smaller features. This one is listed in [the SO answer which serves as de facto documentation for this topic](https://stackoverflow.com/questions/25336528/dc-js-listening-for-chart-group-render#25351686), which I'll double check when I fix this soon.

**Comment by Dakdeniz** [comment]
Just adding filterAll function with the same functionality of _table.redraw works.
```javascript
_table.filterAll = function() {
            _dt.clear()
                .rows.add(_dimension.top(Infinity))
                .draw();
        }; ```

**Comment by YendorMc** [comment]
HI 
I added Dakdeniz's code in dc.datatable.js after _table.redraw @ line 117.  

I now get get "DataTables warning: table id=DataTables_Table_0 - Cannot reinitialise DataTable. For more information about this error, please see http://datatables.net/tn/3"

Any thoughts as to what else can be the issue?

**Comment by gordonwoodhull** [comment]
@YendorMc, I can’t think of any reason why this code would work for redraw but not for filterAll. 

Just to confirm, do you mean that you get the warning when you click the ‘Reset All’ link? Or some other time?

**Comment by YendorMc** [comment]
Yes - I do get the warning when I do "Reset All' (it's an alert), or also when I have a preset "View" when I set filters on 'presets'
e.g.
```js
function filterit(filters){
	dc.filterAll();

	if (filters == "Region 1"){
		isStateSelect.filter([["AR ","TX ","AL "]])
	}
	else if (filters == "Region 2"){
		isStateSelect.filter([["FL ","GA "]])
	}
    dc.renderAll();
 };
```
It does work after the popup warning, but would be nice to figure out the annoying popup warning.

On a side note - dc.js is absolutely amazing, Gordon!
